### PR TITLE
chore: update Eclipse related gas oracles

### PR DESCRIPTION
**Pull request by tkporter**
### Description

Updates some gas oracles.

Before change:
- All exchange rates were set to 10, gas prices 1, and token decimals 9, which surprisingly wasn't so inaccurate
- Costs before were as follows:
  - Solana -> Eclipse
    - On Solana, IGP paid 0.000431 SOL * $154 = $0.066374
    - On Eclipse delivery, relayer 0.00001263 ETH * 2604 = $0.03288 plus ATA payer 0.000020264 ETH * 2604 = $0.05276
  - Eclipse -> Solana
    - On eclipse, igp paid 0.000268 ETH * 2604 = $0.6978
    - on solana delivery, relayer 0.0017926 SOL * 154 = 0.27 plus ata payer 0.00203928 SOL * 154 = 0.314
  - Eclipse -> Ethereum
    - on eclipse, igp paid 0.000248339 ETH * 2604 = 0.6466
    - on Ethereum delivery, relayer 0.002222450816005406 ETH * 2604 = 5.78
  - Ethereum -> Eclipse
    - on ethereum, IGP paid 0.00086103 ETH * 2604 = $2.24
    - on eclipse delivery, same as above solana
- Costs after:
  - Solana -> Eclipse: now charges approx 33 cents
  - Eclipse -> Solana: now charges approx $1.20
  - Eclipse -> Ethereum: now charges approx $10
  - Ethereum -> Eclipse: unchanged

Exchange rates and gas prices are more or less taken from looking at exchange rates, applying a premium, and figuring out approx gas prices on SVM by looking at amount paid divided by compute units used

### CODE REVIEW

**Comment by paulbalaji** [comment]
do any changes need to be applied to `gasPrices.json` as well? https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/mainnet3/gasPrices.json

**Comment by tkporter** [comment]
@paulbalaji nah I'd view this as totally separate for now until we build some closer tooling integrations

**Review by daniel-savu — APPROVED** [review]
I think these were not considered because the runbook suggested leaving the entry empty. Should we change that?
https://www.notion.so/hyperlanexyz/Chain-Addition-EVM-HaaS-Runbook-fb76fa57cc104660b3403eec967afbd2?pvs=4#977c1466bba0430a8ce3303880293149

File: rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "domain": 1,
+        "gasOracle": {
+            "type": "remoteGasData",
+            "tokenExchangeRate": "15000000000000000000",
```
Comment (daniel-savu) [inline rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json:6]: isn't ETH the native token on eclipse as well? Not sure what this represents

File: rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            "gasPrice": "10000000000",
+            "tokenDecimals": 18
+        }
+    },
+    {
+        "domain": 1399811149,
+        "gasOracle": {
+            "type": "remoteGasData",
+            "tokenExchangeRate": "887000000000000000",
```
Comment (daniel-savu) [inline rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json:15]: sol / eth = $145 / $2456 = 0.05895865237, or is this not how it's computed?

File: rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json (hunk 1 above)
Comment (tkporter) [reply rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json:6]: this is 1.5, which is the premium we charge on EVM

File: rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json (hunk 2 above)
Comment (tkporter) [reply rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json:15]: similar deal - this is a 50% premium on approx that value

**Pull request merged**